LuaMenu/widgets/gui_battle_room_window.lua

```diff
@@ -1291,25 +1291,25 @@ local function InitializeSetupPage(mainWindow, pageConfig, nextPage, selectedOpt
 	}
 	nextButton:Hide()
 
+	advButton = Button:New {

```

Comment:
New users should not press Advanced and this UI is somewhere I expect to funnel many new users. It should be as empty as possible.